Marlin/src/feature/host_actions.cpp

```diff
@@ -142,7 +144,7 @@ void host_action(const char * const pstr, const bool eol) {
         break;
       case PROMPT_PAUSE_RESUME:
         msg = PSTR("LCD_PAUSE_RESUME");
-        queue.inject_P(PSTR("M24"));
+        gcode.process_subcommands_now_P(PSTR("M24"));
```

Comment:
Since `host_response_handler` may be called from within the serial UART interrupt, it is not safe to simply execute a G-code handler here. Due to the likely scenarios for this action, it is safe to use the injection buffer.